packaging/nuget/tools/init.ps1

```diff
@@ -33,14 +33,19 @@ try {
 
 	Set-ItemProperty -Path $platformKeyPath -Name "NuGetUser" -Value "true" | Out-Null
 
-
-    $url = "http://particular.net/download-the-particular-service-platform?version=$packageVersion" 
-    $url = $url.ToLowerInvariant(); 
-
-    if($dte){
-	    $dte.ExecuteCommand("View.URL", $url)
-    }
+	Write-Verbose 'Reporting first time install and version information to www.particular.net. This call does not collect any personal information. For more details, see the License Agreement and the Privacy Policy available here: http://particular.net/licenseagreement. Subsequent NuGet installs or updates will not invoke this call.' -verbose
+	$url = 'https://particular.net/api/ReportFirstTimeUsage'
+	$postData  = New-Object System.Collections.Specialized.NameValueCollection
+	$postData.Add("version", $packageversion)
+	$wc = New-Object System.Net.WebClient
+	$wc.UseDefaultCredentials = $true
+	$wc.UploadValuesAsync($url,"post", $postdata)
 } 
 Catch [Exception] { 
 	Write-Warning $error[0]
 }
+finally {
+	if ($wc){
+	$wc.Dispose()
```

Comment:
As far as I know, the call will execute asynchronously, even after the package manager is done and back to the console. I tested this by running the backend locally and set a breakpoint. The nuget install completed and got back to the prompt but the code was still invoked and waiting at the breakpoint I set.  cc / @gbiellem